1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,226 @@
+For in-depth troubleshooting, a new cmdlet Get-Error to provide detailed error information.
+
+__Key Design Considerations__
+
+1. To reduce confusion and improve debugging success for new and occasional users, error messages
+   should call WriteErrorLine to produce a simplified message for interactive CLI users.
+
+- The error message will contain a prefix as described:
+
+    - If the error is an Exception, it prefixes with Exception:.
+    - If the error has InvocationInfo.MyCommand, it prefixes the command.
+    - If the error has InvocationName, CategoryInfo.Category, or CategoryInfo.Reason, the message
+      will prefix these.
+    - Only if none of those exist does it actually use Error:.
+
+- Simplified error message syntax from 'Message'. (See graphic below)
+
+```powershell
+PS C:\> Get-Childitem -Path c:\notreal
+Get-Childitem: Cannot find path ‘C:\notreal’ because it does not exist
+```
+
+2. To improve script debugging for advanced module builders and scripters, a refactored error view
+   will be displayed. If the error is not from a script or parser error, then it's a single line
+   error message. Otherwise, you get a multiline error message that contains the error and a pointer
+   and error message showing where the error is in that line.
+
+- A new property **ErrorAccentColor** is added to support changing the accent color of the error
+  message. If the terminal doesn't support Virtual Terminal, then vt100 color codes are not used.
+
+```powershell
+PS C:\> .\MyScript.ps1
+Get-ChildItem: in /Users/Username/GitHub/Errorview/script.test.ps1
+Line |
+15   | Get-ChildItem -Path c:\notreal
+     |                     ^^^ Cannot find path 'C:\notreal' because it does not exist.
+    * Help: Additional help information provided here
+```
+
+- An example is in the image below:
+
+![Message and Analytic](.\RFC00XX-Update-Error-View.png)
+
+3. A new cmdlet `Get-Error` will produce comprehensive detailed view of the fully qualified error,
+   including nested inner exceptions.
+
+- Rendering is recursive for nested objects for Exceptions, InvocationInfo, and Arrays otherwise it
+  uses ToString().
+- Members that are empty or null are not shown.
+- Indentation uses 4 spaces for nested objects
+- A new FormatAccentColor is introduced to highlight property names from their values. This can be
+  used later to add accents to tables and list formatting.
+- Removed some commented out unneeded code from ConciseView.
+
+- `Get-Error` will provide the following:
+
+    - Display the newest Error ($Error[0]) – default behavior
+    - Accept Pipeline input – support $error[1] |
+    - Option for the Newest X errors in the session (This will be aliased with Last)
+
+- `Get-Error` syntax
+
+```powershell
+Get-Error  [-InputObject <psobject>] [-Newest <Int32>] [-All] [<CommonParameters>]
+```
+
+First parameter set
+
+- Newest
+
+    + Datatype: int32
+    + specifies one or more of the newest errors to display
+    + Not required
```

Comment:
my assumption is that this is defaulted to `1`

Reply:
@JamesWTruher Yes